package/docker/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ghcr.io/foundry-rs/foundry:nightly-aeba75e4799f1e11e3daba98d967b83e286b0c4a as FOUNDRY
+
+FROM ubuntu:jammy
+
+ENV TZ=America/Chicago
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY --from=FOUNDRY /usr/local/bin/forge /usr/local/bin/forge
+COPY --from=FOUNDRY /usr/local/bin/anvil /usr/local/bin/anvil
+COPY --from=FOUNDRY /usr/local/bin/cast /usr/local/bin/cast
+
+RUN    apt-get update        \
+    && apt-get upgrade --yes \
+    && apt-get install --yes \
+        curl                 \
+        git                  \
+        make                 \
+        python3              \
+        python3-pip          \
+        sudo                 \
+        wget
+
+RUN echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+RUN curl -sSL https://install.python-poetry.org | POETRY_HOME=/usr python3 - --version 1.3.2
```

Comment:
This can probably be dropped.

Reply:
Recorded: https://github.com/runtimeverification/evm-semantics/issues/1813 (I don't want to have to go through another round of testing for something that's working now)